src/data/partnersData.tsx

```diff
@@ -0,0 +1,43 @@
+import Translate from "@docusaurus/Translate";
+import React from "react";
+
+import huawei from "../../static/img/supporters/huawei.png";
+import daocloud from "../../static/img/supporters/DaoCloud.png";
+import DSK from "../../static/img/supporters/DSK.png";
+
+const PartnersList = [
+  {
+    image: huawei,
+    title: "Huawei",
+    webURL: "https://www.huaweicloud.com/intl/en-us/",
+    description: (
+      <Translate id="partners.huawei.description">
+        Huawei Cloud is a leading cloud service provider in China and Asia
+        Pacific.
+      </Translate>
+    ),
+  },
+  {
+    image: daocloud,
+    title: "DaoCloud",
+    webURL: "https://www.daocloud.io/en/",
+    description: (
+      <Translate id="partners.daocloud.description">
+        Build a more resilient and elastic hybrid-multi-cloud platform using Karmada.
+      </Translate>
+    ),
+  },
+  {
+    image: DSK,
+    title: "DSK",
+    webURL: "https://www.unionbigdata.com/",
+    description: (
+      <Translate id="partners.DSK.description">
+        Use Karmada to build a intelligent and distributed container platform with multi-policy and multi-region.
```

Comment:
@pandaoknight Please confirm it.

Reply:
Sorry for the delayed response.
The descriptions in both languages are clear. 

Please change the company name from **DSK** to **Unionbigdata**; this has been confirmed with our business team.

Reply:
Got it. Working on it. See https://github.com/karmada-io/website/pull/549.